### PR TITLE
Implementing a Role for the oawaiver Rails application

### PR DESCRIPTION
**Pull request by jrgriffiniii**
Resolves https://github.com/pulibrary/oawaiver/issues/24

### CODE REVIEW

File: roles/oawaiver/molecule/default/converge.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  become: true
+  pre_tasks:
+    - name: update cache
+      apt:
+        update_cache: true
+        cache_valid_time: 600
+  tasks:
+    - name: "Include mariadbserver"
+      include_role:
+        name: "mariadbserver"
```
Comment (jrgriffiniii) [inline roles/oawaiver/molecule/default/converge.yml:19]: This was needed in order to ensure that the creation of MariaDB users and databases are properly tested. Unfortunately, I am still struggling to resolve one error with the Molecule test suite.

**Comment by jrgriffiniii** [comment]
This is now working, but I am afraid that smoke testing this for the `staging` environment is going to be blocked by https://github.com/pulibrary/oawaiver/pull/23 and https://github.com/pulibrary/oawaiver/pull/31

File: roles/oawaiver/molecule/default/verify.yml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    package:
+      name: "{{ item }}"
+      state: present
+    check_mode: true
+    register: pkg_status
+    loop:
+      - libpq-dev
+      - python-psycopg2
+      - python3-psycopg2
+      - postgresql-client-10
```
Comment (kayiwa) [inline roles/oawaiver/molecule/default/verify.yml:16]: shouldn't this be `postgresql-client-13` ?

**Review by kayiwa — COMMENTED** [review]
change the postgresql-client version in your tests

**Review by kayiwa — APPROVED** [review]
OK

**Pull request merged**